src/Test.UnitTests.WorkItems/Test.UnitTests.WorkItems.csproj

```diff
@@ -10,6 +10,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net48</TargetFrameworks>
```

Comment:
>net48 [](http://example.com/codeflow?start=76&length=5)

To simulate the issue, we need to change to targetFramework 48. It will happen every time.

Reply:
As a side note, I also tried to add net48 for all the other test projects to enhance our testing capabilities BUT the pipeline is timing out. So, we have to investigate why is that happening.

Reply:
Also included a simple and focused test to prevent us from facing this issue again.